articles/user-guide/libraries/standard/math.md

```diff
@@ -1,20 +1,21 @@
 ---
+title: Math in standard Q# libraries
+description: Learn about classical math functions in Q# standard libraries that are used with built-in data types.
 author: cgranade
-description: Learn about the classical math functions in the Q# standard libraries that are used with the built-in data types.
 ms.author: chgranad
-ms.date: 08/29/2021
+ms.date: 06/27/2022
 ms.service: azure-quantum
 ms.subservice: qsharp-guide
-ms.topic: conceptual
 no-loc: ['Q#', '$$v']
-title: Math in the Q# standard libraries
 uid: microsoft.quantum.libraries.overview.math
+ms.topic: conceptual
+ms.custom: kr2b-contr-experiment
 ---
 
 # Classical Mathematical Functions #
 
-These functions are primarily used to work with the Q# built-in data types `Int`, `Double`, and `Range`.
+These functions primarily work with Q# built-in data types `Int`, `Double`, and `Range`.
 
-For example, the <xref:Microsoft.Quantum.Random.DrawCategorical> has signature `(Double[] => Int)`. It takes a categorical distribution specified by a list of probablities, such as an array of doubles as input, and returns a randomly-selected index into the array as an `Int`.
+As an example, the <xref:Microsoft.Quantum.Random.DrawCategorical> has the signature `(Double[] => Int)`. It takes a categorical distribution specified by a list of probablities. An array of doubles can be entered as an input, and then return a randomly-selected index into the array as an `Int`.
```

Comment:
This feels really awkward now?

Reply:
Hi Cassandra, I understand it may feel a little awkward. 

The sentence was taken from passive voice to active voice. I recommend that we keep the change as the style guide advises that we use active voice wherever possible.